Makefile

```diff
@@ -32,12 +34,19 @@ docker:
 	@rm VERSION
 
 .PHONY: generate
-generate:
+generate: .tmp/protoc/bin/protoc
 	@go install google.golang.org/protobuf/cmd/protoc-gen-go@a709e31e5d12
 	@go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
-	@go install github.com/jhump/protoreflect/desc/sourceinfo/cmd/protoc-gen-gosrcinfo
+	@go install github.com/jhump/protoreflect/desc/sourceinfo/cmd/protoc-gen-gosrcinfo@v1.14.1
```

Comment:
This line wouldn't run without this explicit version. The `v1.14.1` tag for this repo is already in `go.mod`, however there are some transitive deps of this package that are not otherwise used by this repo, so they had no entries in `go.sum`. This gets it working again.

Reply:
Explicit deps FTW